.travis.yml

```
@@ -8,5 +8,5 @@ install:
   - nvm use 6
   - npm install --global configurable-http-proxy
 script:
-  - python runner.py
-  - python runner.py --ws
+  - asv machine --yes --machine=travis
+  - asv run --python=python --machine=travis
```

Try changing this to `asv dev`. I think it might give us better feedback when there are errors.

```suggestion
  - asv dev --python=python --machine=travis
```